pkg/sql/maxcompute_submitter.go

```diff
@@ -28,7 +30,9 @@ import (
 	"sqlflow.org/sqlflow/pkg/sql/codegen/pai"
 )
 
-type paiSubmitter struct{ *defaultSubmitter }
+var tarball = "job.tar.gz"
```

Comment:
Why rename `pai_submitter` to `maxcompute_submitter`? I think we should keep the name `maxcompute` always stands for a database driver. And, PAI as one of the execution platform.

Reply:
It's just because of submitting a PAI job using `odpscmd` instead of the PAI command.

Reply:
I revert the renaming, can renaming it in the future if we want to submit PAI job in varients ways as as PAI command line.